Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -40,6 +40,14 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
     OWSAssert(thread);
 
     NSMutableArray<OWSTableItem *> *items = [@[
+        [OWSTableItem itemWithTitle:@"Perform 100 random actions"
+                        actionBlock:^{
+                            [DebugUIMessages performRandomActions:100 thread:thread];
+                        }],
+        [OWSTableItem itemWithTitle:@"Perform 1,000 random actions"
```

Comment:
I've made this stress test even more stressful by sometimes performing multiple changes within a single transaction.